src/main/java/org/primefaces/component/linkbutton/LinkButton.java

```diff
@@ -45,30 +49,34 @@
         return ComponentUtils.getUIParams(this);
     }
 
-    public String resolveStyleClass() {
-        String icon = getIcon();
-        Object value = getValue();
-        String styleClass = "";
+    public String resolveStyleClass(FacesContext context) {
```

Comment:
I know we have other resolveStyleClass in component, but I think it makes more sense to have that in renderers

Reply:
I was wondering why this was here.  I agree I should move it to the renderer.

Reply:
moved into renderer